teuthology/orchestra/console.py

```diff
@@ -35,6 +35,18 @@ def __init__(self, name, ipmiuser, ipmipass, ipmidomain, logfile=None,
         self.has_credentials = all(
             [self.ipmiuser, self.ipmipass, self.ipmidomain]
         )
+        self.conserver_master = config.conserver_master
+        self.conserver_port = config.conserver_port
+        conserver_client_found = subprocess.Popen(
+            'which console',
```

Comment:
Not 100% sure about this.  1, design: is having the client enough to turn this on?  2, mechanism: is which universal, and always an external binary, and should we allow it to happen from any path?